2-mandatory/7-eligible-students.js

```diff
@@ -14,10 +14,14 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
  */
 
-function eligibleStudents(attendances) {}
+function eligibleStudents(attendances) {
+  return attendances
+    .filter((student) => student.attendance >= 8)
+    .map((student) => student.name);
+}
 
```

Comment:
Good job on chaining `filter()` and `map()` methods to return the required output.